modules/cluster/src/main/java/io/fluo/cluster/runner/ScanOptions.java

```diff
@@ -40,6 +40,10 @@
   @Parameter(names = {"-h", "-help", "--help"}, help = true, description = "Prints help")
   public boolean help;
 
+  @Parameter(names = {"-esc", "--escape-non-ascii"}, help = true,
+      description = "Hex encode non ascii bytes")
+  public boolean hexEncNonAscii = false;
```

Comment:
you could change the option to be `-noesc` or `--no-escape-non-ascii` so you don't have to set it to false